extensions/adobe/experience/content-telemetry/webasset.schema.json

```diff
@@ -0,0 +1,102 @@
+        },
+        "repo:lastModifiedDate": {
+          "type": "string",
+          "format": "date-time",
+          "meta:usereditable": false,
+          "examples": ["2004-10-23T12:00:00-06:00"],
+          "description": "The server date and time when the resource was last modified in the repository, such as when a new version of an asset is uploaded or a directory's child resource is added or removed. The date time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
+        },
+        "xdm:url": {
+          "title": "Url",
+          "type": "string",
+          "format": "uri",
+          "description": "The absolute URL of the asset."
+        },
+        "xdm:manualTags": {
+          "type": "array",
+          "title": "Tags",
+          "items": {
+            "title": "Tag",
+            "type": "string",
+            "description": "The list of manually assigned tags for the asset"
+          },
+          "description": "The `manualTags` property is used to track human-assigned descriptive phrases like keywords or tags of an asset."
+        },
+        "xdm:predictedTags": {
+          "type": "array",
+          "title": "Predicted Tags",
+          "items": {
+            "$schema": "http://json-schema.org/draft-06/schema#",
+            "title": "Predicted tag",
+            "description": "The `Predicted tag` schema describes a tag in a list of tags, with specific importance relative to other tags in the same list.",
+            "properties": {
+              "xmp:value": {
+                "title": "Value",
+                "description": "The tag itself. A short description of the content of the webelement.",
+                "type": "string"
+              },
+              "dc:language": {
+                "type": "string",
+                "pattern": "^(((([A-Za-z]{2,3}(-([A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-([A-Za-z]{4}))?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-([0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(x(-[A-Za-z0-9]{1,8})+))?)|(x(-[A-Za-z0-9]{1,8})+)|((en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang)))$",
+                "description": "Language of the keyword. The locale code should follow [RFC BCP 47](https://tools.ietf.org/html/bcp47)",
+                "examples": ["en-US", "de-CH"]
+              },
+              "xdm:confidence": {
```

Comment:
Can you tighten the semantic meaning of confidence, by projecting property name as confidenceScore.